chalicelib/check_setup.json

```diff
@@ -1558,5 +1558,44 @@
                 }
             }
         }
+    },
+    "indexer_server_status": {
```

Comment:
Is this info not possible to arrange as part of a decorator? Why is it separated from the function name? (This doesn't have to be solved for this PR, just curious.)

Reply:
Yeah, this info is all required and packaged separately from the code. See the first few methods in `check_utils.py`.